mlrun/runtimes/__init__.py

```diff
@@ -91,6 +93,7 @@ class RuntimeKinds(object):
     dask = "dask"
     job = "job"
     spark = "spark"
+    igzsparkclient = "igzsparkclient"
```

Comment:
@urihoenig need a simpler runtime name, e.g. `remote-spark` (i also assume in future we can extend it to work against other Spark services, but even if not it needs to reflect the usage)

Reply:
@yaronha - This runtime is practically kubejob with some Iguazio Spark specific methods.
It was originally sparkclient, but we realized that it has nothing that isn't Iguazio specific (and it won't change in the future).
I can change it to be igzspark if you prefer a shorter name (I haven't seen us using '-' in a runtime name).

Reply:
i think the essence vs the other runtime is that it works against a service (remote), both run over igz, so its confusing

Reply:
The other spark runtime is specific for spark operator.
As opposed to this runtime, it's not specific for iguazio.
It would be better to rename it to be sparkop or sparkk8s, but this will require massive modifications of tests & demos.
I think that it would be best to modify it in 0.7.0 and make sure we align all our demos.

Anyway, the igzspark relies on a very specific iguazio configuration which won't have any other code to share with other runtimes.

Reply:
@yaronha